feedthebeast/feedthebeast.json

```diff
@@ -1,55 +1,49 @@
 {
-  "type": "minecraft",
+  "type": "minecraft-java",
   "display": "FTB Launcher - Minecraft",
   "install": [
     {
       "type": "download",
-      "files": "${link}"
+      "files": "https://api.modpacks.ch/public/modpack/${modpack_id}/${modpack_version}/server/linux"
     },
     {
       "type": "command",
       "commands": [
-        "unzip -u FTB*.zip"
+        "chmod u+x serverinstall_${modpack_id}_${modpack_version}",
+        "./serverinstall_${modpack_id}_${modpack_version} --auto --noscript"
       ]
     },
     {
       "type": "move",
-      "source": "FTB*.zip",
-      "target": "ZIP.zip"
-    },
-    {
-      "type": "command",
-      "commands": [
-        "rm ZIP.zip",
-        "chmod u+x FTBInstall.sh",
-        "chmod u+x ServerStart.sh",
-        "chmod u+x settings.sh",
-        "./FTBInstall.sh"
-      ]
+      "source": "forge-*.jar",
+      "target": "ForgeServer.jar"
     },
     {
       "type": "writefile",
       "text": "eula=${eula}",
       "target": "eula.txt"
     },
-    {
-      "type": "writefile",
-      "text": "export MAX_RAM=\"${memory}M\"",
-      "target": "settings-local.sh"
-    },
     {
       "type": "writefile",
       "text": "server-ip=${ip}\nserver-port=${port}\nmotd=${motd}\n",
       "target": "server.properties"
     }
   ],
   "run": {
+    "program": "java",
+    "arguments": [
+      "-server",
+      "-XX:+UseG1GC",
+      "-XX:+UnlockExperimentalVMOptions",
+      "-Xmx${memory}M",
+      "-Xms3072M",
```

Comment:
This should also be ${memory}, not a hardcoded value

Reply:
You are right. I have commited that change